class RetryError(Exception):
    pass


class ExpiredError(Exception):
    pass
